Solutions/Menes.Hosting.AspNetCore/Menes/Internal/OpenApiHttpResponseResult.cs

```diff
@@ -315,7 +317,15 @@ private void BuildHeaders(HttpResponse httpResponse, OpenApiResponse response)
                         convertedValue = this.ConvertValue(header.Value.Schema, value);
                     }
 
-                    httpResponse.Headers.Add(header.Key, new Microsoft.Extensions.Primitives.StringValues(convertedValue));
+                    // If the input value was a string, it will have been returned as if it were a serialized JSON element.
```

Comment:
Have you seen https://learn.microsoft.com/en-us/dotnet/api/microsoft.net.http.headers.headerutilities?view=aspnetcore-7.0

Specifically

https://learn.microsoft.com/en-us/dotnet/api/microsoft.net.http.headers.headerutilities.removequotes?view=aspnetcore-7.0#microsoft-net-http-headers-headerutilities-removequotes(microsoft-extensions-primitives-stringsegment)

Reply:
Good call!